src/Processor/Path/Symfony/PathProcessor.php

```diff
@@ -53,6 +54,7 @@ public function process(Path $path): array
      */
     private function processRoute(SymfonyRoute $route, SymfonyRoutePath $path): array
     {
+        Assert::isNonEmptyList($route->getMethods());
```

Comment:
We should throw `InvalidArgumentException` like in `process()` with a custom message.